src/app/page.tsx

```diff
@@ -25,6 +25,7 @@ export default function Home() {
               priority
             />
             <div>aaa</div>
+            <div>しゅんぺ～</div>
```

Comment:
```suggestion
            <div>こしゅん</div>
```
@taka0910 
こしゅんの方がいいと思うので、変更してもらえますか？